### ISSUE TITLE
07BF0 sensor ID

### ISSUE THREAD

**Issue by rootless4real** [That-Guy-Jack/HP-ILO-Fan-Control#10]
You can add DL80 Gen9 to the list.  It works very well.  Thank you, thank you, thank you!

There is one issue I've been unable to resolve.  There is a sensor id 07BF0 that is tied to the on-board drive controller - B140i.  I cannot address that sensor.  All the rest are 1-2 digits.

Is there a way to allow it to address that sensor?  Or any other known work-around?

Thank you, thank you.

**Comment by That-Guy-Jack** [comment]
Sounds intresting ill have a look what mine reports, 
any chance you can send what the console says when trying to access it?

**Comment by rootless4real** [comment]
What it looks like in the console is this:

`</>hpiLO-> fan info g`

`GROUPINGS`
`0: FASTEST Output: 150  [07FB00 08 09 10 17 19 21 22 27 28 34 35 39 40 49 50 51 52 53 54 60*]`
`1: FASTEST Output: 150  [03 04 07FB00 08 09 10 17 21 22 27 28 34 35 38 39 40 49 50 51 52 53 54 60*]`
`2: FASTEST Output: 150  [01 03 04 07FB00 08 09 10 11 13 15 18 19 21 22 23 24 27 29 30 31 34 36 37 38 39 40 41 42 43 44 49 50 51 52 55 56 57 58 60*]`
`3: FASTEST Output: 150  [01 03 04 05 06 07FB00 08 09 10 11 13 15 18 19 20 21 22 23 24 27 29 30 31 34 36 37 38 39 40 41 42 43 44 49 50 51 52 55 56 57 58 60*]`
`4: FASTEST Output: 150  [02 05 06 07FB00 08 12 14 16 18 20 24 25 26 30 31 32 33 36 37 43 44 45 46 47 48 49 50 55 56 57 58 60*]`
`5: FASTEST Output: 150  [02 05 06 07FB00 08 12 14 16 20 24 25 26 30 32 33 36 43 44 45 46 47 48 49 50 55 56 60*]`

I tried using the 07FB00 (and every variation I could think of) to address it when using 'fan pid xx lo/hi 3500/5000'.  I don't have what was returned (something like 'address doesn't exist').

What I found to work was to turn off that sensor using, `fan t 07 off`.

That worked miracles.
